book/src/install.md

```diff
@@ -1,180 +1,224 @@
+accepts the new settings on first use.
 
-```
-sudo dnf copr enable varlad/helix
-sudo dnf install helix
-```
+If you are using a version of Nix without flakes enabled,
+[install Cachix CLI](https://docs.cachix.org/installation) and use
+`cachix use helix` to configure Nix to use cached outputs when possible.
+ 
+## macOS
 
-### Void Linux
+### Homebrew Core
 
-```
-sudo xbps-install helix
+```sh
+brew install helix
 ```
 
 ## Windows
 
-Helix can be installed using [Scoop](https://scoop.sh/), [Chocolatey](https://chocolatey.org/)
+Install on Windows using [Scoop](https://scoop.sh/), [Chocolatey](https://chocolatey.org/)
 or [MSYS2](https://msys2.org/).
 
-**Scoop:**
+### Scoop
 
-```
+```sh
 scoop install helix
 ```
 
-**Chocolatey:**
+### Chocolatey
 
-```
+```sh
 choco install helix
 ```
 
-**MSYS2:**
-
-Choose the [proper command](https://www.msys2.org/docs/package-naming/) for your system from below:
-
-  - For 32 bit Windows 7 or above:
-
-```
-pacman -S mingw-w64-i686-helix
-```
-
-  - For 64 bit Windows 7 or above:
-
-```
-pacman -S mingw-w64-x86_64-helix
-```
+### MSYS2
 
-  - For 64 bit Windows 8.1 or above:
+For 64-bit Windows 8.1 or above:
 
-```
+```sh
 pacman -S mingw-w64-ucrt-x86_64-helix
 ```
 
-## Build from source
+## Building from source
 
-```
+Clone the repository:
+
+```sh
 git clone https://github.com/helix-editor/helix
 cd helix
-cargo install --path helix-term --locked
 ```
 
-This will install the `hx` binary to `$HOME/.cargo/bin` and build tree-sitter grammars in `./runtime/grammars`.
-
-If you are using the musl-libc instead of glibc the following environment variable must be set during the build
-to ensure tree sitter grammars can be loaded correctly:
+Compile from source:
 
+```sh
+cargo install --path helix-term --locked
 ```
-RUSTFLAGS="-C target-feature=-crt-static"
-```
-
 
-Helix also needs its runtime files so make sure to copy/symlink the `runtime/` directory into the
-config directory (for example `~/.config/helix/runtime` on Linux/macOS). This location can be overridden
-via the `HELIX_RUNTIME` environment variable.
+This command will create the `hx` executable and construct the tree-sitter
+grammars either in the `runtime` folder, or in the folder specified in `HELIX_RUNTIME`
+(as described below). To build the tree-sitter grammars requires a c++ compiler to be installed, for example `gcc-c++`.
 
-| OS                   | Command                                          |
-| -------------------- | ------------------------------------------------ |
-| Windows (Cmd)        | `xcopy /e /i runtime %AppData%\helix\runtime`    |
-| Windows (PowerShell) | `xcopy /e /i runtime $Env:AppData\helix\runtime` |
-| Linux / macOS        | `ln -s $PWD/runtime ~/.config/helix/runtime`     |
+> 💡 If you are using the musl-libc instead of glibc the following environment variable must be set during the build
+> to ensure tree sitter grammars can be loaded correctly:
```

Comment:
```suggestion
> to ensure tree-sitter grammars can be loaded correctly:
```

Reply:
> The current link looks good to me, I don't think we'll need an anchor

I have removed the TOC link to the main heading, and also made the same correction to the TOC  in `usage.md`.

It goes against the way the marksman language server creates TOCs, but is no problem, I assume this is what you meant, it does seem better.

Reply:
Everything should be sorted now :) Please squash it all down when you commit it, I have not been very disciplined with my commit naming and won't try to squash it myself in case I jinx it. I have learned a lot about Git doing this pull request, thanks for all your help!